tests/Traits/DataHelpers/ProductHelper.php

```diff
@@ -264,4 +264,14 @@ public function createCustomer(string $customerId): void
             Context::createDefaultContext()
         );
     }
+
+    public function getBasicVariantData(array $overrides = []): array
+    {
+        return array_merge([
+            'stock' => 10,
+            'active' => true,
+            'tax' => ['name' => '9%', 'taxRate' => 9],
+            'price' => [['currencyId' => Defaults::CURRENCY, 'gross' => 15, 'net' => 10, 'linked' => false]]
```

Comment:
I think it's easier to read when each entry has it's own line, no real need to change.